packages/vuetify/src/util/helpers.ts

```diff
@@ -596,6 +596,22 @@ export function focusChild (el: Element, location?: 'next' | 'prev' | 'first' |
   }
 }
 
+export function getTabbableChild (el: Element, location?: 'next' | 'prev') {
+  const targets = ['button', '[href]', 'input:not([type="hidden"])', 'select', 'textarea', '[tabindex]']
+    .map(s => `${s}:not([disabled])`)
+    .join(', ')
+  const children = [...el.querySelectorAll(targets)] as HTMLElement[]
```

Comment:
This is already exists in a function: https://github.com/vuetifyjs/vuetify/pull/17674/files#diff-6daf9735657f26888ecc2f70a2fa7b67dfef7600ab7275e0ae89b111dd461c6dR567

Reply:
I had to add an optional arg to focusableChildren to use it for what I needed it for (without the `:not(tabindex="-1")` condition).